core/tx_pool.go

```diff
@@ -273,6 +280,9 @@ type TxPool struct {
 	initDoneCh      chan struct{}  // is closed once the pool is initialized (for tests)
 
 	changesSinceReorg int // A counter for how many drops we've performed in-between reorg.
+
+	enableTxLifecycles bool
```

Comment:
we can move it to `TxPoolConfig` and provide a cmd argument

Reply:
why support this config is because don't want the signer node to record this metric. I think it's no necessary to add this to config.

Reply:
for example, you can follow the idea of https://github.com/scroll-tech/go-ethereum/blob/feat/calculate_tx_duration/cmd/utils/flags.go#L503-L507 and https://github.com/scroll-tech/go-ethereum/blob/feat/calculate_tx_duration/miner/miner.go#L61

Reply:
yes, I know what you mean. But I want to say that just using the `--mine` flag can distinguish the signer node, so I don't add a flag to it.